pkg/collector/corechecks/containers/docker/unbundled_events.go

```diff
@@ -27,22 +27,26 @@ func newUnbundledTransformer(hostname string, types []string) eventTransformer {
 	return &unbundledTransformer{
 		hostname:            hostname,
 		collectedEventTypes: collectedEventTypes,
+		eventTransformer:    eventTransformer,
```

Comment:
nit: thoughts on renaming this to something like `bundledTransformer` like you have in your previous PR for kubernetes events? i think it would make it a bit easier to understand